src/bindings/python/fluxacct/accounting/user_subcommands.py

```diff
@@ -269,6 +269,61 @@ def view_user(conn, user, parsable=False, cols=None, list_banks=False):
         raise ValueError(f"view-user: {exc}")
```

Comment:
commit message is out of date, default is no longer for JSON and there is no `--parsable` option?